administrator/language/de-DE/com_contact.ini

```diff
@@ -58,7 +52,7 @@ COM_CONTACT_FIELD_CONFIG_TABLE_OF_CONTACTS_DESC="Diese Einstellungen gelten für
 COM_CONTACT_FIELD_CONFIG_VCARD_LABEL="vCard"
 COM_CONTACT_FIELD_CONTACT_SHOW_CATEGORY_LABEL="Kontaktkategorie"
 COM_CONTACT_FIELD_CONTACT_SHOW_LIST_LABEL="Kontaktliste"
-COM_CONTACT_FIELD_CREATED_BY_ALIAS_LABEL="Autoralias"
+COM_CONTACT_FIELD_CREATED_BY_ALIAS_LABEL="Autor-Alias"
```

Comment:
```suggestion
COM_CONTACT_FIELD_CREATED_BY_ALIAS_LABEL="Autoralias"
```

Reply:
Wir lassen den mal so, wird in MOD_ARTICLES_CATEGORY_FIELD_AUTHORALIAS_LABEL in mod_articles_category.ini auch so verwendet.